### ISSUE TITLE
rootDir option causes an error

### ISSUE THREAD

**Issue by Quramy** [teppeis/typescript-simple#40]
If my tsconfig.json has `rootDir` option, TypeScriptSimple causes the following error.

```
TypeError: Cannot read property 'text' of undefined
    at TypeScriptSimple.toJavaScript (/Users/yosuke/git/typescript-simple/index.js:161:28)
    at TypeScriptSimple.compile (/Users/yosuke/git/typescript-simple/index.js:71:25)
    at Object.<anonymous> (/Users/yosuke/git/typescript-simple/repro.js:3:5)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:457:10)
    at startup (node.js:136:18)
    at node.js:972:3
```

I could reproduce it with the following code.

```js:repro.js
var TypeScriptSimple = require('./index').TypeScriptSimple;
var tss = new TypeScriptSimple({outDir: 'built/', rootDir: 'src/'}, false);
tss.compile('var x = 123;', 'src/file.ts');
```

I think it's because that output file name includes the value of `rootDir` in `toJavaScript` method.
* correct output file name(TS Language Service created): `built/file.js` 
* actual output file name in this method: `built/src/file.js`

**Comment by teppeis** [comment]
Fixed in #41 and v5.0.2 has been released.
Thanks! :+1:

**Comment by Quramy** [comment]
Thanks for your quick review and release :)
